src/CVRF/cvrf_eval.c

```diff
@@ -253,7 +253,6 @@ void cvrf_export_results(struct oscap_source *import_source, const char *export_
 		cvrf_element_to_dom("DiscoveryDate", cvrf_vulnerability_get_discovery_date(vuln), vuln_node);
 		cvrf_element_to_dom("ReleaseDate", cvrf_vulnerability_get_release_date(vuln), vuln_node);
 		cvrf_element_to_dom("CVE", cvrf_vulnerability_get_cve_id(vuln), vuln_node);
-		cvrf_element_to_dom("CWE", cvrf_vulnerability_get_cwe_id(vuln), vuln_node);
```

Comment:
Yes. This element has been replaced with a structure (it has content and an attribute according to CVRF specs)